students/AY1617S2/tanMinRongSamson/TanMinRongSamson-Resume.md

```diff
@@ -0,0 +1,82 @@
+Samson Tan
+==========
+<p align="center">
+  <a href="mailto:samson.tmr@gmail.com">Email</a> |
+  <a href="linkedin.com/in/samsontmr">LinkedIn</a> |
+  <a href="github.com/samsontmr">GitHub</a> |
+  <a href="bit.ly/samsontanresume">Resume</a>
+</p>
+
+Education
+---------
+**BComp. Computer Science + University Scholars Programme** <br>
+National University of Singapore | Singapore
+August 2015 - Present
+
+**Diploma with Merit in Digital Visual Effects** <br>
+Nanyang Polytechnic | Singapore <br>
+April 2010 - May 2013 <br>
+
+Experience
+----------
+
+**Undergraduate Student Researcher / Research Assistant** <br>
+National University of Singapore | Singapore <br>
+January 2017 - Present <br>
+***Responsibilities***
+*   Conduct research analysis such as computerized topical/sentiment/semantic network analysis
```

Comment:
Add a blank line before starting a bullet list. Otherwise it will not render properly after converting to html by GitHub Pages (although it looks fine when viewed as a markdown page on GitHub)